### DISCUSSION TITLE
Usage in other projects

### DISCUSSION THREAD

**Discussion by Naegolus** [c-ares/c-ares#924] [Q&A] [answered]
Hi,

according to [c-ares.org](https://c-ares.org/#who-is-c-ares)
> Please let us know if you use it!

## Usage

Our small project built around the [Processing()](https://github.com/NoOrientationProgramming/ProcessingCore) class is using cares:
[DnsResolving()](https://github.com/NoOrientationProgramming/ProcessingCommon/blob/main/DnsResolving.cpp) ([doc](https://github.com/NoOrientationProgramming/ProcessingCommon/blob/main/DnsResolving.md))
[HttpRequesting()](https://github.com/NoOrientationProgramming/ProcessingCommon/blob/main/HttpRequesting.cpp) ([doc](https://github.com/NoOrientationProgramming/ProcessingCommon/blob/main/HttpRequesting.md))

## Version
```
Package: libc-ares-dev
Version: 1.18.1-1ubuntu0.22.04.3
```

## Feedback

Works like a charm. Thx!

Best regards,
Joe

**Comment by bradh352** [comment, marked as answer]
Feel free to send a PR to add yourself to https://github.com/c-ares/c-ares.github.io/blob/main/index.md :)

Btw, c-ares 1.18.1 is super old.  I'd say 70% of the code has been replaced since that version...

**Comment by Naegolus** [comment]
Wow. Thx. Sure!

**Reply by Naegolus** [reply]
Done
